src/Bonsai.ML.LinearDynamicalSystems/Bonsai.ML.LinearDynamicalSystems.csproj

```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Title>Bonsai.ML.LinearDynamicalSystems</Title>
+    <Description>A Bonsai package for implementing the Kalman Filter using python.</Description>
+    <Author>Nicholas Guilbeault</Author>
+    <Copyright>Copyright ©  2023</Copyright>
```

Comment:
As we discussed, it might be easier to maintain a separate `AUTHORS` file with the list of significant package contributors which can be updated easily over time.

I was checking a few examples and I really liked the approach used by the [flutter](https://github.com/flutter/flutter/blob/master/AUTHORS) project, where they have a clear format and expectations for the file, and also an invitation for proactive editing of the file via PRs which would help to keep the maintenance of the file more distributed and scalable.